test/index.js

```diff
@@ -86,60 +98,77 @@ describe('Bananas', () => {
                                 request: {
-                                    id: updates[2].request.id,
-                                    received: updates[2].request.received,
-                                    elapsed: updates[2].request.elapsed
+                                    id: updates[3].request.id,
+                                    received: updates[3].request.received,
+                                    elapsed: updates[3].request.elapsed
                                 },
                                 error: {
                                     message: 'Uncaught error: boom',
-                                    stack: updates[2].error.stack
+                                    stack: updates[3].error.stack
                                 }
                             },
                             {
                                 event: 'response',
-                                timestamp: updates[3].timestamp,
+                                timestamp: updates[4].timestamp,
                                 host: Os.hostname(),
                                 tags: ['test'],
                                 path: '/',
                                 query: {},
                                 method: 'get',
                                 request: {
-                                    id: updates[3].request.id,
-                                    received: updates[3].request.received,
-                                    elapsed: updates[3].request.elapsed
+                                    id: updates[4].request.id,
+                                    received: updates[4].request.received,
+                                    elapsed: updates[4].request.elapsed
                                 },
                                 code: 500,
                                 error: {
                                     statusCode: 500,
                                     error: 'Internal Server Error',
                                     message: 'An internal server error occurred'
                                 }
-                            },
-                            {
-                                event: 'server',
-                                timestamp: updates[4].timestamp,
-                                host: Os.hostname(),
-                                tags: ['test', 'some', 'tags'],
-                                data: {
-                                    message: 'oops',
-                                    stack: updates[4].data.stack,
-                                    data: 42
-                                }
                             }
                         ]);
 
-                        Wreck.post = orig;
+                        expect(new Date(updates[0].timestamp)).to.be.between(timeBeforeBananasRegister, new Date());
+                        expect(new Date(updates[1].timestamp)).to.be.between(timeBeforeServerStart, new Date());
```

Comment:
@hueniverse - cool, I didn't know lab has the `onCleanup` callback as a second argument.
also appreciated the double meaning of https://github.com/hueniverse/bananas/commit/36a8c9c828c73b4dd5f3323c63ce4bb023bbf5fd [Cleanup for #15]